documentation/docs/decentralized_identifiers/overview.md

```diff
@@ -9,68 +9,69 @@ keywords:
 
 :::
 
 Services are URIs that point to more information about the identity. This could be something as simple as a website for an organizational identity. These services are publicly available for all to read and should therefore not contain Personal Identifiable Information (PII) in the case of human identities.
 
-## Why use DIDs?
+## Why Use DIDs?
 
-DIDs allow any subject to have a unique identifier, that they can prove ownership of and at the same time provide a way to send them encrypted messages. The Identity is Self-Sovereign, meaning the subject is in control of when the identity is created but also destroyed.
+DIDs allow any subject to have a unique identifier, that they can prove ownership of and, at the same time, provide a way to send them encrypted messages. The Identity is Self-Sovereign, meaning the subject is in control of when the identity is created but also destroyed.
 
-DIDs become more interesting in combination with Verifiable Credentials, which will be covered in a later section. In essence, Verifiable Credentials (VCs) are signed statements by trusted third parties about a certain identity. The signer, or Issuer, is referenced by the DID and so is the subject, often called the Holder. The Holder controls a copy of this statement and share it with other parties, the Verifiers, that can verify the statement and check which party made the statement, without having to ask the Issuer. Instead, they can verify the signature of the Issuer by checking the Issuers DID Document. This whole setup puts Holders back in control over their own data, but also makes the data much more trustworthy as it has become verifiable.
+DIDs become more interesting in combination with Verifiable Credentials, which will be covered in a later section. Essentially, Verifiable Credentials (VCs) are signed statements by trusted third parties about a certain identity. The signer, or Issuer, is referenced by the DID and so is the subject, often called the Holder. The Holder controls a copy of this statement and share it with other parties, the Verifiers, that can verify the statement and check which party made the statement, without having to ask the Issuer. Instead, they can verify the signature of the Issuer by checking the Issuers DID Document. This whole setup puts Holders back in control over their own data, but also makes the data much more trustworthy as it has become verifiable.
 
-## Why use IOTA Identity over other implementations?
+## Why Use IOTA Identity Over Other Implementations?
 
-IOTA Identity is a framework to implement Self-Sovereign Identities on IOTA. Inherently, IOTA provides some unique features that have a major impact on the usability of the framework.
+IOTA Identity is a framework to implement SSI on IOTA. Inherently, IOTA provides unique features that have a major impact on the usability of the framework.
```

Comment:
Since every page can be page one, in theory, if it's the first time we introduce a term, an acronym may not be clear to the user.
```suggestion
IOTA Identity is a framework to implement Self-Sovereign Identities (SSI) on IOTA. Inherently, IOTA provides unique features that have a major impact on the framework's usability.
```